pkg/version/parse_test.go

```diff
@@ -0,0 +1,38 @@
+package version
```

Comment:
can we test `version` as an external package? ie make this `version_test`?

Reply:
done